modules/programs/fzf.nix

```diff
@@ -195,8 +195,10 @@ in {
     # Note, since fzf unconditionally binds C-r we use `mkOrder` to make the
     # initialization show up a bit earlier. This is to make initialization of
     # other history managers, like mcfly or atuin, take precedence.
+    # Still needs to be initialized after oh-my-zsh (order 800), otherwise
+    # omz will take precedence.
     programs.zsh.initContent =
-      mkIf cfg.enableZshIntegration (mkOrder 200 zshIntegration);
+      mkIf cfg.enableZshIntegration (mkOrder 801 zshIntegration);
```

Comment:
nit (and also I'm not a maintainer): if OMZ has prio 800, maybe we should pick a slightly lower prio (larger number) here (e.g. 808 or 810), to leave the door open for slotting things in between?

I hope we wouldn't have to actually use that functionality, of course 😅 

Thanks so much for making this PR!

Reply:
good point, ill update to 810